ProcessMaker/processmaker
# FOUR-8177 Nayra Container Service S3

The following PR includes:

- Nayra Service implementation
- Queue Manager Service implementation
  - Driver for RabbitMq
  - Drive for Kafka
- Improvement of start event
- Improvement of complete Task
- Improvement of engine endpoints

## For Cloud Ops

- Once the installation is complete (Core and packages) run the following command to update the routes cache:

```
APP_RUNNING_IN_CONSOLE=false php artisan route:cache
```

## How to test

Run the ProcessMaker as usual, some interesting places to check the difference:

- Click on `+ REQUEST` button to start an already existing process
- Choose a process and start it
- Open the Task fill the screen and complete it

...
